Code/GraphMol/SubstructLibrary/substructLibraryTest.cpp

```diff
@@ -1,12 +1,32 @@
+//  Copyright (c) 2017-2019, Novartis Institutes for BioMedical Research Inc.
```

Comment:
Please add `  RDLog::InitLogs();` at the top of main() in this file so that the various status/error messages actually end up showing up in the console.
I should have noticed this when I did the original SubstructLibrary review and didn't. sorry.

Reply:
Sure!